src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const checkPresence = (value: string) => {
+    return value.toLowerCase().includes(query.toLowerCase());
+  };
```

Comment:
great decision to separate this logic from filtering 🔥 
We can improve naming though
```suggestion
  const doesQueryMatch = (value: string) => {
    return value.toLowerCase().includes(query.toLowerCase());
  };
```